Project.toml

```
@@ -10,7 +10,7 @@ TimeStruct = "f9ed5ce0-9f41-4eaa-96da-f38ab8df101c"
 
 
 [compat]
-JuMP = "^1.5"
-SparseVariables = "^0.7.3"
-TimeStruct = "^0.8"
-julia = "^1.9"
+JuMP = "1.5"
+SparseVariables = "0.7.3"
+TimeStruct = "0.9"
+julia = "1.9"
```

1.10 is now LTS, so might as well bump this requirement?

```suggestion
julia = "1.10"
```